xbmc/Util.cpp

```diff
@@ -2147,7 +2135,19 @@ ExternalStreamInfo CUtil::GetExternalStreamDetailsFromFilename(const std::string
         continue;
       }
 
-      name += " " + (*it);
+      if (info.language.empty())
+      {
+        std::string langTmp(*it);
+        std::string langCode;
+        // try to recognize language
+        if (g_LangCodeExpander.ConvertToISO6392B(langTmp, langCode))
```

Comment:
define the variable `langTmp` seem not needed,
can you remove it? and use `*it`

Reply:
good catch. done